compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/bir/codegen/split/JvmCreateTypeGen.java

```diff
@@ -576,6 +576,14 @@ private AnonTypeHashInfo createLabelsForAnonTypeHashSwitch(Set<BIRTypeDefinition
         return new AnonTypeHashInfo(hashes, labels, labelFieldMapping);
     }
 
+    public void generateRefTypeConstants(List<BIRTypeDefinition> typeDefs, SymbolTable symbolTable) {
```

Comment:
The type-reference type constants need to be created for all reference type definitions, as they are referred to when called from another module.